TC39 short survey results

captainbrosset opened the pull request:
Live preview of the report and analysis: https://github.com/web-platform-dx/developer-research/blob/tc39/mdn-short-surveys/2025-04-16-tc39-proposals/README.md

captainbrosset committed:
TC39 short survey results

captainbrosset committed:
Decorator comments

captainbrosset committed:
Improved formatting

captainbrosset committed:
todos

captainbrosset committed:
todo

captainbrosset committed:
analysis for all proposals

tidoust reviewed (commented):
See some typos and comments inline.

The interpretation highlights commenters' sentiments but does not seem to do much with it. Anything that stands out of the ordinary? More generally speaking, do the overall results more or less match what you were expecting?

Could you also update the table in the main repo README to link to this survey?

captainbrosset commented on mdn-short-surveys/2025-04-16-tc39-proposals/README.md:
```diff
@@ -0,0 +1,269 @@
+
+## Interpretation
+
+The following interpretations are proposed by [Patrick Brosset](https://github.com/captainbrosset), PM at Microsoft Edge.
```
```suggestion
The following interpretations are proposed by [Patrick Brosset](https://github.com/captainbrosset), product manager at Microsoft Edge.
```

captainbrosset committed:
Apply suggestions from code review

Co-authored-by: François Daoust <fd@w3.org>

captainbrosset committed:
Add TC39 survey to README

captainbrosset committed:
typo

captainbrosset committed:
Overall learnings

captainbrosset commented:
@tidoust please take a look.

I removed the sentiment analysis. I realized I really didn't have time to dig more into it. I'll leave it to folks to look into the CSV file for interesting bits of data. So, instead of leaving an un-interpreted set of negative/positive tables, I just removed the tables.

I did add an overall learnings section at the end though.

captainbrosset commented on README.md:
```diff
@@ -11,22 +11,23 @@ In practice, the WebDX Community Group regularly discusses surveys targeted at w
-| [DevSat Survey](vendor-research/browser-support-matrix-survey.pdf)          | Google                  | Mar 2023 | 852     | N/A                                                             | [PDF slides](vendor-research/Web%20Dev%20Sat%20H1%202023.pdf)                                              |
-| [Browser Support Matrix](vendor-research/browser-support-matrix-survey.pdf) | Google                  | Nov 2022 | 233     | N/A                                                             | [PDF slides](vendor-research/browser-support-matrix-survey.pdf)                                            |
-| [Web Browsers](mdn-short-surveys/2022-10-24-browser-support)                | WebDX / MDN             | Oct 2022 | 964     | [CSV](mdn-short-surveys/2022-10-24-browser-support/results.csv) | [Interpretation](mdn-short-surveys/2022-10-24-browser-support/interpretation.md)                           |
+## History of surveys tracked by the group
```
Sorry about the large diff here. This table was formatted with plenty of empty space to try and align the columns like a real table. While this can be nice when tables are narrow, it didn't work for me here, as my screen isn't wide enough for the table to be fully visible, and the wrapping was quite bad.
I don't think it's useful to try and keep the formatting.
